### PR TITLE
#89 refactor project structure and sbt files

### LINKED ISSUE #89: Refactor Project Structure and SBT files
The Project contains some oddities in the SBT structure as well as odd naming conventions. This issue is to go over the project and rectify and normalize these.

### PR DESCRIPTION
**Pull request by TheLydonKing**
Did a few changes that were mentioned by Sasa for sbt formatting and useage. Most Suggestions were minor and implemented in previous commits.

-This branch contains the larger change of refactoring the directory name (Not large in scope but changes may be difficult to review if it was baked into another branch)

Also minor change of removing parent module value and assigning those settings globally.

Closes #89

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -20,6 +20,8 @@ ThisBuild / organization := "za.co.absa.login-service"
 
 lazy val scala212 = "2.12.17"
 
+ThisBuild / name := "login-service"
```
Comment (Zejnilovic) [inline build.sbt:23]: This shouldn't be there

**Review by Zejnilovic — COMMENTED** [review]
Still seeing:
- `publish / skip := true`
- ThisBuild / homepage still commented out

File: build.sbt (hunk 1 above)
Comment (TheLydonKing) [reply build.sbt]: Hi Sasa

Thank you for the review. I've made changes as per the comments. It should look better now?

**Review by dk1844 — APPROVED** [review]
The changes LGTM, just please make sure that we have everything covered from Sasa's list in https://github.com/AbsaOSS/login-service/issues/89

**Review by Zejnilovic — APPROVED** [review]

**Pull request merged**